Doc/library/os.path.rst

```diff
@@ -341,6 +341,14 @@ the :mod:`glob` module.)
    that contains symbolic links.  On Windows, it converts forward slashes to
    backward slashes. To normalize case, use :func:`normcase`.
 
+  .. note::
+      On POSIX systems, in accordance with `IEEE Std 1003.1 2013 Edition; 4.13
+      Pathname Resolution <http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap04.html#tag_04_13>`_,
+      if a pathname begins with two successive characters, the first component
```

Comment:
"begins with two successive characters":  IMHO it would be clearer to write "begins with exactly two slashes".  "Two successive characters" could be interpreted as any characters instead of just slashes, and the implementation checks for exactly 2 slashes (3 or more slashes are replaced by a single slash).

Reply:
Yes, you are right.  "begins with exactly two slashes" it's more clear. I updated the PR. Thanks @ronaldoussoren !